CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+= 0.9.5 =
```

Comment:
Let's call this "<version> - in development" until we decide to release,  so we don't make the mistake of adding more releases than one :)